### PR TITLE
Added sysrst_ctrl to nightly regression

### PR DESCRIPTION
**Pull request by patel-madhuri**
Signed-off-by: Madhuri Patel <madhuri.patel@ensilica.com>

### CODE REVIEW

File: hw/top_earlgrey/dv/top_earlgrey_sim_cfgs.hjson [hunk 1]
```diff
@@ -44,12 +44,12 @@
              "{proj_root}/hw/ip/rom_ctrl/dv/rom_ctrl_sim_cfg.hjson",
              "{proj_root}/hw/ip/rstmgr/dv/rstmgr_sim_cfg.hjson",
              "{proj_root}/hw/ip/rv_timer/dv/rv_timer_sim_cfg.hjson",
-             "{proj_root}/hw/ip/spi_host/dv/spi_host_sim_cfg.hjson",
```
Comment (weicaiyang) [inline hw/top_earlgrey/dv/top_earlgrey_sim_cfgs.hjson:47]: please add this back. I think you probably delete this accidentally when solving the merge conflict.

File: hw/top_earlgrey/dv/top_earlgrey_sim_cfgs.hjson [hunk 2]
```diff
@@ -44,12 +44,12 @@
...
              "{proj_root}/hw/ip/spi_device/dv/spi_device_sim_cfg.hjson",
              "{proj_root}/hw/ip/sram_ctrl/dv/sram_ctrl_main_sim_cfg.hjson",
              "{proj_root}/hw/ip/sram_ctrl/dv/sram_ctrl_ret_sim_cfg.hjson",
              "{proj_root}/hw/ip/uart/dv/uart_sim_cfg.hjson",
              "{proj_root}/hw/ip/usbdev/dv/usbdev_sim_cfg.hjson",
+             "{proj_root}/hw/ip/sysrst_ctrl/dv/sysrst_ctrl_sim_cfg.hjson",
```
Comment (sriyerg) [inline hw/top_earlgrey/dv/top_earlgrey_sim_cfgs.hjson:52]: Please maintain alphabetical order as suggested above.

File: hw/ip/sysrst_ctrl/dv/sysrst_ctrl_sim_cfg.hjson [hunk 3]
```diff
@@ -49,17 +49,14 @@
     {
       name: sysrst_ctrl_smoke
       uvm_test_seq: sysrst_ctrl_smoke_vseq
-      reseed: 5
-    }
+   }
```
Comment (sriyerg) [inline hw/ip/sysrst_ctrl/dv/sysrst_ctrl_sim_cfg.hjson:52]: nit: spacing

**Review by weicaiyang — APPROVED** [review]
LGTM

**Pull request merged**